server/conn.go

```diff
@@ -492,12 +494,15 @@ func (cc *clientConn) addMetrics(cmd byte, startTime time.Time, err error) {
 // It also gets a token from server which is used to limit the concurrently handling clients.
 // The most frequently used command is ComQuery.
 func (cc *clientConn) dispatch(data []byte) error {
+	span := opentracing.StartSpan("server.dispatch")
```

Comment:
There are commands we don't care which are noises too.
How about tracing only `handleQuery` and `handleStmtExecute`?

Reply:
There won't be any noises currently, because we haven't  trace requests other than `handleQuery`...
Trace `dispatch` has a potential benefit that we can find whether requests are blocked by rate limiter:

```
	token := cc.server.getToken()
	defer func() {
		cc.server.releaseToken(token)
		span.Finish()
	}()
```